cv_next/app/upload/page.tsx

```diff
@@ -48,24 +48,28 @@ const InputRow = ({
 };
 
 export default function Page() {
-  const [catagoryId, setCatagoryId] = useState<number[]>([]);
+  const [categoryId, setCategoryId] = useState<number[]>([]);
   const [description, setDescription] = useState("");
   const [link, setLink] = useState("");
   const { showError } = useError();
 
+  const [isLinkTouched, setIsLinkTouched] = useState(false);
+  const [isDescriptionTouched, setIsDescriptionTouched] = useState(false);
+  const [isCategoryTouched, setIsCategoryTouched] = useState(false);
```

Comment:
It might be easier to use `react-hook-form`, I already started incorporating it into our codebase in #204. It gives you these kinds of options as well (know when the field has been touched/"dirtied").

Reply:
Updated this